### ISSUE TITLE
N2 enabled: Amiibo Backup

### ISSUE THREAD

**Issue by burndive** [HiddenRamblings/TagMo#302]
Testing with 3b39585:
Using dir /Amiibo2 as working directory, with no bin files present.

1. Backup amiibo when N2 Elite is enabled in Settings
Menu -> Amiibo Backup -> (Scan amiibo) -> Error: Reading Amiibo data failed!
This works as expected when N2 Elite is disabled in Settings.

2. Backup N2 Bank
Menu -> Amiibo Backup -> (Select slot number and scan N2) -> N2 Bank 1 data is saved, regardless of which Bank is selected, and regardless of which Bank is Active
When N2 is enabled in settings, this works as expected to backup the Active Bank.

**Comment by burndive** [comment]
Another interesting data point regarding strange characters at the end of Amiibo nicknames in bin files: 

I think the Wii U handles these differently than the Switch (though the Switch displays old nicknames correctly). I saved a backup of my Mario amiibo with both the official N2 Elite app and TagMo. The Switch displays the nickname as "Mario TZ", but the bin file has "Mario TZ켐". 

When I saved this bin to an N2 Bank and used the Switch to change the name to "Mario N2", and then obtained a bin file, the nickname is just "Mario N2".
